dask/dataframe/io/tests/test_parquet.py

```diff
@@ -597,3 +596,18 @@ def test_parquet_select_cats(tmpdir):
     assert list(rddf.columns) == ['ints']
     rddf = dd.read_parquet(fn)
     assert list(rddf.columns) == list(df)
+
+
+@pytest.mark.parametrize('compression,', ['default', None, 'gzip', 'snappy'])
+def test_writing_parquet_with_compression(tmpdir, compression, engine):
+    fn = str(tmpdir)
+
+    if engine == 'fastparquet' and compression == 'snappy':
+        pytest.importorskip('snappy')
+
+    df = pd.DataFrame({'x': ['a', 'b', 'c'] * 10, 'y': [1, 2, 3] * 10})
+    ddf = dd.from_pandas(df, npartitions=3)
+
+    ddf.to_parquet(fn, compression=compression, engine=engine)
+    out = dd.read_parquet(fn, engine=engine)
+    assert_eq(out, ddf)
```

Comment:
`assert_eq` assumes one of the arguments is a pandas dataframe. You probably meant `df` instead of `ddf`.

Reply:
Note that you can run the tests locally instead of waiting for travis.

```
# assumes all dependencies are already installed
py.test dask/dataframe/io/tests/test_parquet.py -k test_writing_parquet_with_compression
```

Reply:
All parquet tests fail on my machine. I'll change the data frame input parameter and hope it goes ok

Reply:
> All parquet tests fail on my machine.

Please check that you have the proper versions of pyarrow/fastparquet, and their dependencies installed.